build/azure-pipelines.yml

```diff
@@ -34,8 +34,9 @@ stages:
         propsFile: 'Firely.Fhir.Packages.props'
         packageArtifacts: true
         restoreDependencies: true
-        nuGetServiceConnections: GitHubPackageGetFeed
-        nuGetSources: --source https://nuget.pkg.github.com/FirelyTeam/index.json
+        # we shouldn't use a private feed since we are public repo
```

Comment:
Looking at the template, I see some inconsistency:
```
nuGetServiceConnections: #required when restoreDependies = true
nuGetSources: #required when restoreDependies = true
 ```
and digging further, in https://github.com/FirelyTeam/azure-pipeline-templates/blob/193a8796cd4b68e784a6f33a8699b01036383a1f/restore.yml: 
```
  - task: NuGetAuthenticate@1
    inputs:
      nuGetServiceConnections: ${{ parameters.nuGetServiceConnections }}
```
Should we make a check on the nuGetServiceConnections not being empty?

Reply:
I think that's at least nice to do, although it currently works without it being present. 
Also, and `nuGetSources` should just not be marked as required

Reply:
See: https://github.com/FirelyTeam/azure-pipeline-templates/pull/25